setup-functional-tests.sh

```diff
@@ -0,0 +1,11 @@
+git clone --depth 1 https://github.com/mozilla/addons-server.git
+docker-compose -f addons-server/docker-compose.yml -f docker-compose.override.yml up -d --build
+sleep 20
```

Comment:
You should replace this sleep command with something that polls for doneness (it could be fixed in a future patch). You could write a script that runs a loop until some file exists or waits for `docker images` to show the right image or `docker-compose ps` to show the right process.